From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aleksei Kuzmin <alkuzmin@microsoft.com>
Date: Thu, 20 Sep 2018 17:46:10 -0700
Subject: no_stack_dumping.patch


diff --git a/content/app/content_main_runner_impl.cc b/content/app/content_main_runner_impl.cc
index 4caa237a5c89f9ff8d84af7dff9b850fbd89b301..9afdfe74512b943d43a3b751396316bc0da78f2a 100644
--- a/content/app/content_main_runner_impl.cc
+++ b/content/app/content_main_runner_impl.cc
@@ -777,7 +777,7 @@ int ContentMainRunnerImpl::Initialize(const ContentMainParams& params) {
           return nullptr;
         }));
 
-#if !defined(OFFICIAL_BUILD)
+#if 0
 #if defined(OS_WIN)
     bool should_enable_stack_dump = !process_type.empty();
 #else
-- 
2.16.2

